sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90

```diff
@@ -39,34 +39,44 @@ PROGRAM lake_frac
     REAL, PARAMETER :: d2r = acos(-1.0) / 180.0
     REAL, PARAMETER :: r2d = 180.0 /acos(-1.0) 
     REAL, PARAMETER :: pi = acos(-1.0) 
-    REAL*8, PARAMETER :: gppdeg = 119.99444445
-    REAL*8, PARAMETER :: delta = 1.0 / 119.99444445
+!   REAL*8, PARAMETER :: gppdeg = 119.99444445
+!   REAL*8, PARAMETER :: delta = 1.0 / 119.99444445
+    REAL*8, PARAMETER :: gppdeg = 120.0
+    REAL*8, PARAMETER :: delta = 1.0 / 120.0
```

Comment:
will this change for gppdeg an delta 119.99444445 to 120  affect geolocations?

Reply:
No.